projects/packages/jetpack-mu-wpcom/src/features/wpcom-sidebar-notice/wpcom-sidebar-notice.js

```diff
@@ -67,17 +69,37 @@ const wpcomShowSidebarNotice = () => {
 			wp.ajax.post( 'wpcom_dismiss_sidebar_notice', {
 				id: sidebarNotice.dataset.id,
 				feature_class: sidebarNotice.dataset.featureClass,
-				_ajax_nonce: wpcomSidebarNotice.dismissNonce,
+				_ajax_nonce: wpcomSidebarNoticeData.dismissNonce,
 			} );
 			sidebarNotice.remove();
 
 			// Record dismiss event in Tracks.
-			wpcomSidebarNoticeRecordEvent( wpcomSidebarNotice.tracks?.dismiss );
+			wpcomSidebarNoticeRecordEvent( wpcomSidebarNoticeData.tracks?.dismiss );
 		} else {
 			// Record click event in Tracks.
-			wpcomSidebarNoticeRecordEvent( wpcomSidebarNotice.tracks?.click );
+			wpcomSidebarNoticeRecordEvent( wpcomSidebarNoticeData.tracks?.click );
 		}
 	} );
 };
 
-document.addEventListener( 'DOMContentLoaded', wpcomShowSidebarNotice );
+const wpcomFetchSidebarNotice = async () => {
+	try {
+		const response = await fetch(
+			`${ wpcomSidebarNoticeConfig.ajaxUrl }?action=wpcom_fetch_sidebar_notice&nonce=${ wpcomSidebarNoticeConfig.nonce }`
+		);
+
+		if ( ! response.status === 200 ) {
+			return;
+		}
```

Comment:
Nitpick (non-blocking): I found this syntax a bit confusing.
```suggestion
		if ( response.status !== 200 ) {
			return;
		}
```